project/app/src/main/java/org/owntracks/android/ui/preferences/PreferencesFragment.java

```diff
@@ -226,6 +226,7 @@ private void populateScreenNotification(PreferenceScreen screen) {
         PreferenceCategory background = getCategory(R.string.preferencesCategoryNotificationBackground);
         screen.addPreference(background);
         addSwitchPreference(background, Preferences.Keys.NOTIFICATION_EVENTS, R.string.preferencesNotificationEvents, R.string.preferencesNotificationEventsSummary, R.bool.valNotificationEvents);
+        addSwitchPreference(background, Preferences.Keys.NOTIFICATION_SHOW_SELF, R.string.preferencesNotificationShowSelf, R.string.preferencesNotificationShowSelfSummary, R.bool.valNotificationShowSelf);
```

Comment:
I'm not sure what you mean. Could you elaborate that a bit?

Reply:
> I think we can drop the whole preference screen. Events only work on MQTT mode and all display options are already present through the system notification channels. Details can then easily be changed through the configuration editor

Do you mean that this behaviour should only be switchable via the config editor and not a toggle on the config screen? If so, what should the default be? I can see good arguments for both enabling and disabling this, so a preference screen toggle seems reasonable to me.

Reply:
The whole notification preference screen is redundant in HTTP mode. 
I'd get rid of it in MQTT mode too because the notification channels can easily be changed through the system GUI or for some people through the config editor if they bother to read the documentation.

Reply:
Do you mean through the Android OS Notifications settings? The problem with that is, that you can only deactivate all notifications and it's not possible to be selective (e.g. only hide my own notifications). On top of it, you then get the "An app is running the background" notifications by the OS and if you hide that too, then you won't know in the future if another app is running in the background without your knowledge.

If this is not what you mean, would you mind pointing us in the right direction? I couldn't find anything in the Booklet relating to this apart from the above.

Reply:
For most users it should be enough to either enable or disable the ongoing or events notification through the OS notification channel settings. Disabling it in the app can lead to unwanted side effects. 

Hence I suggest to remove the GUI notification preferences. If a user wants to tweak the defaults nevertheless, the configuration editor can then be used to e.g. disable notifications for own events.

Reply:
I figured out now what you meant by "configuration file" which wasn't immediately clear (for reference: https://owntracks.org/booklet/tech/json/).

I personally think, however, that this is a very complicated way of configuring the app. I also believe that disabling notifications via the OS is a poor way of handling this, since it doesn't give fine control over which notifications one wants to hide and which not. But I'll leave this decision to you.

I would suggest first adding this new setting and merging the change. If you guys want to delete the entire settings dialog completely afterwards, I'll leave that to your discretion. I will update this PR to reflect your previously requested changes, and getting rid of the settings can be done in a separate PR.

Reply:
Btw: You don't have to edit the file by hand. There's a build in configuration editor for all config keys unter Preferenes>Configuration Management>Editor ;)

But you're right. Let's get this PR merged and everthing else sorted later.